service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/generated/GeneratedOrder.java

```diff
@@ -26,4 +30,14 @@
     private final String courtName;
     @JsonSerialize(contentConverter = BasicChildConverter.class)
     private final List<Element<Child>> children;
+    private String removalReason;
+
+    @JsonIgnore
+    public boolean isRemovable() {
+        return BLANK_ORDER.getLabel().equals(type);
```

Comment:
I think it would be better if getLabel is not used bu there is dedicated flag on the order telling us that information. 

Something like:
`
GeneratedOrderType.fromType(type).isRemovable();
`
thoughts?

Reply:
It would be nice to have an internal flag that keeps the order type, I will have a look into it

Reply:
Actually no, your way with a few tweaks might be good

Reply:
👍